public/index.html

```diff
@@ -10,198 +10,15 @@
-      width: 3rem;
-      transition: margin 2s ease;
-    }
-
-    marquee .fries-dude {
-      width: 1rem;
-    }
-
-
-    * {
-      font-family: "Noto Sans Vai";
-    }
-
-    .card {
-      box-shadow: 0 1px 5px grey;
-      border-radius: 8px;
-      overflow: hidden;
-      max-width: 20rem;
-      position: relative;
-    }
-
-    .card>h3 {
-      text-align: center;
-      text-decoration: underline;
-      font-weight: normal;
-      margin: 15px;
-    }
-
-    .card button.refresh {
-      border: none;
-      background-color: transparent;
-      padding: 0;
-      margin: 15px;
-      cursor: pointer;
-    }
-
-    .card button.refresh img {
-      width: 23px;
-    }
-
-    .card img.food-image {
-      width: 100%;
-    }
-
-    .card>p {
-      padding: 0 15px;
-      text-align: center;
-
-    }
-
-    .refresh-wrapper {
-      text-align: right;
-      position: absolute;
-      width: 100%;
-    }
-
-    .taco-question {
-      display: grid;
-      gap: 1.34rem;
-      grid-template-columns: auto 1fr;
-      width: 100%;
-      max-width: 32rem;
-      margin: 0 auto 1.34rem auto;
-    }
-
-    .taco-question .taco-guy {
-      text-align: right;
-      width: 23.8vw;
-      max-width: 10rem;
-    }
-
-    .taco-question .taco-question {
-      text-align: left;
-    }
-
-    @media screen and (max-width: 350px) {
-      .taco-question {
-        grid-template-columns: 1fr;
-        justify-items: center;
-        gap: 0;
-      }
-
-      .taco-question .taco-guy {
-        width: 10rem;
-        text-align: center;
-      }
-    }
-
-    .speech-bubble {
-      border: 1px solid;
-      padding: 1rem;
-      position: relative;
-    }
-
-    .speech-bubble::before,
-    .speech-bubble::after {
-      content: '';
-      position: absolute;
-      top: 10px;
-      left: -40px;
-      width: 30px;
-      display: block;
-      height: 0px;
-      z-index: 1;
-      border-top: 10px solid #d5878700;
-      border-bottom: 10px solid transparent;
-      border-right: 10px solid #000000;
-    }
-
-    .speech-bubble::after {
-      width: 46px;
-      border-right-color: #ffffff;
-      top: 10px;
-      left: -55px;
-    }
-
-    @media screen and (max-width: 350px) {
-
-      .speech-bubble::before,
-      .speech-bubble::after {
-        display: none;
-      }
-    }
-
-    input[type=range] {
-      -webkit-appearance: none;
-      width: 100%;
-      height: 10px;
-      border-radius: 5px;
-      background: #d3d3d3;
-      margin-bottom: 1rem;
-    }
-
-    input[type=range]::-webkit-slider-thumb {
-      -webkit-appearance: none;
-      appearance: none;
-      width: 49px;
-      height: 23.3px;
-      background: url('mustachetaco.svg');
-      cursor: pointer;
-    }
-
-    input[type=range]::-moz-range-thumb {
-      width: 49px;
-      height: 23.3px;
-      border: 0;
-      background: url('mustachetaco.svg');
-      cursor: pointer;
-    }
-  </style>
+  <link rel="stylesheet" href="/styles.css">
```

Comment:
Need a relative link, but great that you've moved into its own file!
```suggestion
  <link rel="stylesheet" href="./styles.css">
```